examples/multi-account-multi-region/main.tf

```diff
@@ -2,11 +2,6 @@
 module "multi_account_multi_region" {
   source = "../../"
 
-  providers = {
-    aws.this = aws.this
-    aws.peer = aws.peer
-  }
-
```

Comment:
You need restore this and than it should works, I think.
https://www.terraform.io/docs/configuration/modules.html#providers

Reply:
sure. for this part, i found there is no different to keep and remove it

But anyway, i will restore it